data/json/mapgen/lab/lab_surface/lab_surface_big_z3.json

```diff
@@ -0,0 +1,258 @@
+        "...........:           #  % %  #           :................................:           #  % %  #           :...........",
+        "...........#           #  % %  #           #................................#           #  % %  #           #...........",
+        "...........:           #  % %  #           :................................:           #  % %  #           :...........",
+        "...........#           #  % %  #           #................................#           #  % %  #           #...........",
+        "...........:           #  % %  #           :................................:           #  % %  #           :...........",
+        "...........#           #  % %  #           #................................#           #  % %  #           #...........",
+        "...........:           #  % %  #           :................................:           #  % %  #           :...........",
+        "...........#           =  % %  =           #................................#           =  % %  =           #...........",
+        "...........#############1 % % E#############................................#############E % % 1#############...........",
+        "...........#           #  % %  #           #................................#           #  % %  #           #...........",
+        "...........:           #  % %  #           :................................:           #  % %  #           :...........",
+        "...........#           #  % %  #           #................................#           #  % %  #           #...........",
+        "...........:           #  % %  #           :................................:           #  % %  #           :...........",
+        "...........#           #  % %  #           #................................#           #  % %  #           #...........",
+        "...........:           #  % %  #           :................................:           #  % %  #           :...........",
+        "...........#           #  % %  #           #................................#           #  % %  #           #...........",
+        "...........:           #  % %  #           :................................:           #  % %  #           :...........",
+        "...........#           #  % %  #           #................................#           #  % %  #           #...........",
+        "...........:           #  % %  #           :................................:           #  % %  #           :...........",
+        "...........#           =  % %  =           #................................#           =  % %  =           #...........",
+        "...........#############E % % 1#############................................#############1 % % E#############...........",
+        "...........#           #  % %  #           #................................#           =  % %  =           #...........",
```

Comment:
I think the north set of doors here on the east wing are probably not intended.

```
#############1 % % E#############
#           =  % %  =           #
:           #  % %  #           :
#           #  % %  #           #
:           #  % %  #           :
#           #  % %  #           #
:           #  % %  #           :
#           #  % %  #           #
:           #  % %  #           :
#           #  % %  #           #
#           #  % %  #           :
#           =  % %  =           #
#############E % % 1#############
```
They open up into sample freezers, and can have hallway clutter in front of them.

![image](https://user-images.githubusercontent.com/11464/54486928-18a5b600-4844-11e9-8896-f95915eefeba.png)

Reply:
Dang. There's always some dumb thing from a previous design that slips through.